ledger-core/v2/application/testwalletapi/handlers.go

```diff
@@ -247,6 +246,7 @@ func (s *TestWalletServer) GetBalance(w http.ResponseWriter, req *http.Request)
 		CallType:       payload.CTMethod,
 		Callee:         *ref,
 		CallSiteMethod: getBalance,
+		Arguments:      insolar.MustSerialize([]interface{}{}),
```

Comment:
can we just check for nil on the other side? not sure if we want to send []byte{0x80}. Or create global var for emptyArguments that could be calculated on compilation